.jenkins/openshift/deploy-master.yaml

```diff
@@ -41,8 +41,8 @@ objects:
   kind: PersistentVolumeClaim
   metadata:
     annotations:
-      volume.beta.kubernetes.io/storage-class: gluster-file
-      volume.beta.kubernetes.io/storage-provisioner: kubernetes.io/glusterfs
+      volume.beta.kubernetes.io/storage-class: netapp-file-standard
+      volume.beta.kubernetes.io/storage-provisioner: kubernetes.io/netapp
```

Comment:
no need to specify the `storageClassName`?

Reply:
we actually moved to ephemeral jenkins so this is not even needed anymore